packages/p2p/src/stores/sendbird-store.ts

```diff
@@ -172,28 +172,28 @@ export default class SendbirdStore extends BaseStore {
         this.setIsChatLoading(true);
         try {
             const group_channel = await this.sendbird_api?.groupChannel.getChannel(this.chat_channel_url ?? '');
-
             if (!group_channel) {
                 this.setHasChatError(true);
             } else {
                 this.setActiveChatChannel(group_channel);
             }
         } catch (error) {
-            this.setChatError();
+            // handle sendbird error
         } finally {
             this.setIsChatLoading(false);
         }
     }
 
     async initialiseOrderMessages() {
         this.setHasChatError(false);
+        this.setIsChatLoading(true);
         try {
             const chat_messages = await this.getPreviousMessages();
             if (chat_messages && chat_messages.length > 0) {
                 this.setChannelMessages(chat_messages.map(msg => convertFromChannelMessage(msg)));
             }
         } catch (error) {
-            this.setHasChatError(true);
```

Comment:
why are we removing this?

Reply:
@farrah-deriv, because it showed the screen shown below

<img width="1198" alt="Screenshot 2023-08-01 at 4 11 56 PM" src="https://github.com/binary-com/deriv-app/assets/104334373/057d806a-cefc-45de-90a3-494f5ad67c24">

when a send bird exception like the one in the screenshot below is thrown

<img width="1605" alt="Screenshot 2023-08-02 at 4 01 28 PM" src="https://github.com/binary-com/deriv-app/assets/104334373/462557c9-6a1c-43af-8a71-0f6918e89da3">

Reply:
isnt that ok tho to have this error?

Reply:
@ameerul-deriv, when those errors occur, the chat still works. In the old implementation, you guys used to log them with console.warn, i replaced it with setChatError, which is why it started appearing when my PR was merged

Reply:
@jim-deriv we can put back `console.warn` from the old implementation so we'll still be informed if there's any error